src/workbench/P5VisualizerTemplate.ts

```diff
@@ -0,0 +1,134 @@
+import {P5Visualizer} from '../visualizers/P5Visualizer'
+import {VisualizerExportModule} from '@/visualizers/VisualizerInterface'
+
+/** md
+# p5 Visualizer Template
+
+[image should go here]
+
+This template can be used as a starting point for designing a p5 visualizer.
+
```

Comment:
Note that the on-line visualizer documentation is generated from these markdown comments. So for this to be a proper template, it should have the same sort of documentation that a "real" visualizer would have, i.e. describe what it does and the keyboard controls. Ideally you can arrange the code so that the markdown describing the keyboard stuff is right next to the keypressed() function where it is implemented. (We want code organization like that so that when someone implements a new key, say, they will see right there to put the documentation in.)

Reply:
I've updated the docblock along these lines. How does it look now?

Reply:
Added some more specific comments now, so resolving this generic one.